build.gradle

```diff
@@ -8,7 +8,7 @@ group 'eu.xenit.postgres'
 version '0.1.0'
 
 def junit_version = "5.7.1"
-def pg_version = "11.12"
+def pg_version = "12.7"
```

Comment:
This is a major upgrade. Do we have a plan to move? Also, the latest 12 is now 12.8. We should probably update docker-postgres first.

Reply:
I included this in the commit by accident. I was planning on looking into building both 11 and 12